bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/discovery/LGWebOSUpnpDiscoveryParticipant.java

```diff
@@ -58,15 +62,24 @@
                 .withProperty(PROPERTY_DEVICE_ID, device.getIdentity().getUdn().getIdentifierString())
                 .withProperty(CONFIG_HOST, device.getIdentity().getDescriptorURL().getHost())
                 .withLabel(device.getDetails().getFriendlyName())
-                .withProperty(PROPERTY_MODEL_NAME, device.getDetails().getModelDetails().getModelName())
+                // .withProperty(PROPERTY_MODEL_NAME,
+                // device.getDetails().getModelDetails().getModelName() + " "
+                // + device.getDetails().getModelDetails().getModelNumber())
+                .withProperty(PROPERTY_MODEL_NAME, device.getDetails().getFriendlyName())
```

Comment:
Because they are not what we expect on this device.
For example, in my case:

- modelname = LG TV
- modelNumber = 1.0

friendlyName remains the same on all devices.